### PR TITLE
Adding Wordpress as a submodule.

### PR DESCRIPTION
**Pull request by hmc-berk**

### CODE REVIEW

File: .gitmodules [hunk 1]
```diff
@@ -1,3 +1,7 @@
 [submodule "app/magento"]
```
Comment (zachChilders) [inline .gitmodules]: We should remove magento.  We can't currently bind both to port 80.

**Review by zachChilders — CHANGES_REQUESTED** [review]
Thanks!  We can't have both right now though.  We'll need to add nginx or another reverse proxy to ensure both containers can be served on the same external port.  Can we remove magento for now?

File: .gitmodules (hunk 1 above)
Comment (hmc-berk) [reply .gitmodules]: Done.

**Review by zachChilders — APPROVED** [review]

**Comment by hmc-berk** [comment]
Added wordpress submodule and removed magento (for now).

**Pull request merged**